apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx

```diff
@@ -37,33 +66,25 @@ export default function VersionUnitDropdowns({
             onChangeVersion={id => updateCourseDetail(VERSION_ID, id)}
           />
         )}
-        {sectionCourse?.unitId && (
-          <span>
-            <div>{i18n.assignmentSelectorCourse()}</div>
-            <select
-              id="uitest-unit-dropdown"
-              value={courseOffering}
-              onChange={updateCourseDetail(UNIT_ID, event.target.value)}
-            >
-              <option key="default" value={noAssignment} />
-              <option key="later" value={decideLater}>
-                {i18n.decideLater()}
-              </option>
-              {courseOffering.map(offering => (
-                <optgroup
-                  key={offering.display_name}
-                  label={offering.display_name}
-                >
-                  {courseOffering?.map(courseOffering => (
-                    <option key={courseOffering.id} value={courseOffering.id}>
-                      {courseOffering.display_name}
-                    </option>
-                  ))}
-                </optgroup>
-              ))}
-            </select>
-          </span>
-        )}
+        {sectionCourse?.versionId !== 0 &&
```

Comment:
Why 0?

Reply:
Excellent question... I copied this logic from the existing component so I'm not sure but I'll go look!

Reply:
I personally would prefer something like `sectionCourse?.versionId &&`. 0 is falsey so it should maintain the existing behavior anyway

Reply:
Updated! I also caught and fixed a small bug. Now selecting a new version clears out an old unit assignment.